src/iw3unp29.f

```diff
@@ -738,11 +727,11 @@ FUNCTION I03O29(NUNIT, OBS, IER)
          REWIND NUNIT
          ISWT = 0
       END IF
- 
+
    10 CONTINUE
 
       IF(NEXT.NE.0)  GO TO 70
- 
+
```

Comment:
Whatever it is you're doing to remove these spaces, if should be done as a separate step from the doxygenation.

That is, do the doxygen without touching these lines, to make review easier, then do another pass when you remove trailing blanks (which I assume is being done by your editor...)

Reply:
I have no idea what's causing it yet. I am using atom.io but it doesn't happen with all the files I edit, which makes it weird. As soon as I figure out what is causing it, I intend to remedy it.

Reply:
OK. ;-) Sometimes editors can be funny!